content/docs/analytics/QuerySuggestions.md

```diff
@@ -0,0 +1,163 @@
+
+
+### Filter by minimum count
+
+It lets you define the minimum count value which means that only those suggestions which have the larger `count` value will get populated in the suggestions index. The default value is set to `1`.
+
+### Filter by minimum hits
+
+By default, the suggestions index only includes queries that return at least five results in the source index. You can configure this limit from the dashboard.
+
+### Blacklist queries
+
+You can define a set of unwanted queries which will be ignored to extract the suggestions.
+
+### Whitelist indices
+
+It allows you to define which index/indices should be considered to populate the suggestions.
+
+### Number of days
+
+By default, we calculate the suggestions for past `30` days which is <strong>configurable</strong>. We recommend to not use a value less than `30` so you have enough data in your index.
+
+### External Suggestions
+
+Since analytics is the only source to populate the `.suggestions` index. When you get started, you'll need some kind of starting data which can be helpful to display the suggestions.
+
+You can define the external suggestions in the JSON format, each suggestion must have the `query` and `count` keys. The value of the `count` key determines the popularity of a particular suggestion.
+You can check the below example of external query suggestions:
+
+```json
+[
+	{
+		"query": "iphoneX",
+		"count": 10000, // scoring parameter
+		"meta": {
+			// define meta properties
+			"image": "https://abc.com/cat.png"
+		}
+	},
+	{
+		"query": "samsung",
+		"count": 700
+	}
+]
+```
+
+## How to query suggestions
+
+We populate the suggestions in `.suggestions` index, to use the query suggestions you just need to use the `.suggestions` index to get the hits.
+
+### Usage Example With Searchbox
+
+```js
+import Searchbase from '@appbaseio/searchbase';
+import searchbox from '@appbaseio/searchbox';
+
+const instance = new Searchbase({
+	index: '.suggestions',
+	credentials: `CLUSTER_CREDENTIALS`,
+	url: 'CLUSTER_URL',
+	size: 5,
+	dataField: 'key',
+});
+searchbox('#git', {}, [
+	{
+		source: searchbox.sources.hits(instance),
+		templates: {
+			suggestion: function(suggestion) {
+				return `<p class="is-4">${suggestion.label}</p>`;
+			},
+			empty: function() {
+				return `<div>No Results</div>`;
+			},
+			loader: function() {
+				return `<div>Loader</div>`;
+			},
+			footer: function({ query, isEmpty }) {
+				return `
+                    <div style="background: #eaeaea; padding: 10px;">Footer</div>
+                `;
+			},
+			header: function({ query, isEmpty }) {
+				return `
+                    <div style="background: #efefef; padding: 10px;">
+                        Hello From Header
+                    </div>
+                `;
+			},
+		},
+	},
+]);
+```
+
+### Usage Example With ReactiveSearch
+
+// TODO: The example is not complete
```

Comment:
@bietkul We will deal with this in a separate issue. I would recommend trying the below demo once. Does this work on partial suggestions terms? It seems that we should have search analyzers enabled on the query suggestions index.